pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -7230,6 +7547,72 @@ def solve_model(model, options=None):
                     results_2 = solver2.solve(
                         model_h, options={"limits/gap": 0.3}, tee=True
                     )
+        elif model.config.hydraulics == Hydraulics.co_optimize_linearized:
+            # Currently, this method is supported for only MINLP solvers accessed through GAMS.
+            # The default solver is Baron and is set to find the first feasible solution.
+            # If the user has SCIP, then the feasible solution from BARON is passed to SCIP, which is then solved to a gap of 0.3.
+            # If the user do not have these solvers installed then it limits the use of co_optimize method at this time.
+            # Ongoing work is geared to address this limitation and will soon be updated here.
```

Comment:
These comments were copied from the section for solving with `Hydraulics.co_optimize` and I don't believe are applicable to `Hydraulics.co_optimize_linearized`; please update.